src/components/Comments/WriteComment.js

```diff
@@ -169,17 +168,17 @@ export const styles = StyleSheet.create({
     marginTop: 7,
     lineHeight: 15,
     fontSize: 12,
-    color: colors.gray
+    color: COLORS.gray8
   }),
   replyToTitle: {
     fontFamily: fonts.inter[600],
     lineHeight: 15,
     fontSize: 12,
-    color: colors.black
+    color: COLORS.black
   },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The text color in `title` and `replyToTitle` styles has been updated. Ensure that these new colors provide sufficient contrast for readability.
```diff
- color: colors.gray
+ color: COLORS.gray8
...
- color: colors.black
+ color: COLORS.black
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->